_about/notes/index.md

```diff
@@ -16,18 +16,20 @@ toc: false
 
 {% include section-index.html docs=site.about %}
 
-The [latest Istio monthly release](https://github.com/istio/istio/releases) is {{site.data.istio.version}} which
-you can download with:
+- The [latest Istio monthly release](https://github.com/istio/istio/releases) is {{site.data.istio.version}} which
+  you can download with(*):
 
-```
-curl -L https://git.io/getLatestIstio | sh -
-```
+  ```
+  curl -L https://git.io/getLatestIstio | sh -
+  ```
 
-The most recent stable release is [0.2.12](https://github.com/istio/istio/releases/tag/0.2.12) which you
-can download with:
+- The most recent stable release is [0.2.12](https://github.com/istio/istio/releases/tag/0.2.12) which you
+  can download with:
 
-```
-curl -L https://git.io/getIstio | sh -
-```
+  ```
+  curl -L https://git.io/getIstio | sh -
+  ```
 
-[Archived documentation for for 0.2.12 release](https://archive.istio.io/v0.2/).
+  [Archived documentation for the 0.2.12 release](https://archive.istio.io/v0.2/docs/).
```

Comment:
Why is /docs needed?

Reply:
because who cares about the front page when looking for docs for a given release ?
it saves (at least) one unnecessary click for our users